src/components/Table/Internal/octable.module.scss

```diff
@@ -525,6 +535,17 @@
       }
     }
 
+    // TODO: handle internal ping styles for other border props e.g. header-border
```

Comment:
what is meant by "ping" here?

Reply:
ping left or right is when the user has scrolled the unfixed columns to either side and the event 'pings' the fixed ones to do some stuff in css.